examples/main.cpp

```diff
@@ -642,6 +643,28 @@ int main(int argc, char **argv)
 
   bool verbose = false;
 
+  // add adiak init here
+  adiak::init(NULL);
+
+  // replace with adiak::collect_all(); once adiak v0.4.0
+  adiak::uid();
+  adiak::launchdate();
+  adiak::launchday();
+  adiak::executable();
+  adiak::executablepath();
+  adiak::workdir();
+  adiak::libraries();
+  adiak::cmdline();
+  adiak::hostname();
+  adiak::clustername();
+  adiak::walltime();
+  adiak::systime();
+  adiak::cputime();
+  adiak::jobsize();
+  adiak::hostlist();
+  adiak::numhosts();
+  adiak::value("compiler", std::string("@RAJAPERF_COMPILER@"));
+
```

Comment:
Shouldn't these function calls be guarded with` #ifdef __ENABLE_ADIAK__` and `#endif` respectively?

Reply:
Yes, good catch! Added the guards.